transforms/builder.js

```diff
@@ -0,0 +1,195 @@
+import registerMethods from "./registerMethods";
+
+const run = (...transformers) => {
+  return function transformer(fileInfo, api, options) {
+    let didReplacement = false;
+    const j = api.jscodeshift;
+    registerMethods(j);
+    const root = j(fileInfo.source);
+
+    transformers.forEach((transformer) => {
+      const result = transformer(fileInfo, api, options, j, root);
```

Comment:
Slightly confusing that this is in a function called `transformer` but is also calling a different function `transformer` here - worth renaming one of them do you think?

Reply:
I renamed the exported function to be `rootTransformer`.